server/neptune/workflows/activities/command/command_builder.go

```diff
@@ -31,12 +31,13 @@ func (c *commandBuilder) Build(_ context.Context, v *version.Version, path strin
 
 	// We add custom variables so that if `extra_args` is specified with env
 	// vars then they'll be substituted.
+	// TODO: configure these to only be set by terraform activities
 	envVars := []string{
 		// Will de-emphasize specific commands to run in output.
 		"TF_IN_AUTOMATION=true",
 		fmt.Sprintf("ATLANTIS_TERRAFORM_VERSION=%s", v.String()),
 		fmt.Sprintf("DIR=%s", path),
-		fmt.Sprintf("TF_PLUGIN_CACHE_DIR=%s", c.terraformPluginCacheDir),
+		fmt.Sprintf("TF_PLUGIN_CACHE_DIR=%s", c.cacheDir),
```

Comment:
If you're trying to make a generic command, these env vars don't really fit.

Reply:
Yep, there is a TODO above this that says I will make this configurable through an argument, so only Terraform is aware of this.

Reply:
went ahead and handled the TODO to show what i meant